---
fixes:
  - |
    pytest plugin: fix for potential ``KeyError`` exceptions in test runs when
    gevent is detected within the environment.
